clock_exercise/TickOfTopAlarmButton.qml

```diff
@@ -0,0 +1,44 @@
+import QtQuick 2.0
+
+Item {
+    id: root
+
+    property bool visualizationOfSelected
+    property bool visualizationOfHover
+
+    width: 34
+    height: 34
+
+    Image {
+        opacity: (root.visualizationOfSelected && !root.visualizationOfHover || !root.visualizationOfSelected && root.visualizationOfHover)? 1:0
+        source: "images/ic-tick-selected.svg"
+
+        Behavior on opacity {
+            NumberAnimation {
+                duration: clockManager.timeOfTheDissolveAnimation
+            }
+        }
+    }
+
+    Image {
+        opacity: (root.visualizationOfSelected && root.visualizationOfHover)? 1:0
```

Comment:
Secondo me in generale possiamo evitarci di tirare in ballo l'operatore ternario quando abbiamo operazioni booleane, tanto in ogni caso si risolvono in 1 e 0 implicitamente